.github/workflows/deployProd.yml

```diff
@@ -17,6 +17,19 @@ env:
 jobs:
   deploy-backend:
     runs-on: ubuntu-latest
+    services:
+      postgres-db:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
```

Comment:
I'm pretty confident this is _not_ the way to set the secret, but I'm not sure where the secrets in Github Actions are set/created. Would appreciate any feedback :)

Reply:
Secrets get set in the repo itself (Settings -> Secrets)

Reply:
I'm having a very slow day - why do we need to configure what looks like a local Postgres container for a deploy?

Reply:
That would be because I forgot to remove this block when we figured out we can just run `apt-get install postgres` instead 🤦 Removed now.